android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -101,6 +101,7 @@ public void run() {
         emptyGroupIdQueue();
         mSwipeRefreshWidget.setRefreshing(false);
```

Comment:
Are these three methods always called together?  Could you just put them in their own method?